keyboards/dztech/dz65rgb/keymaps/matthewrobo/keymap.c

```diff
@@ -0,0 +1,323 @@
+#include QMK_KEYBOARD_H
+
+enum dz65rgb_layers {
+	_QWERTY,
+	_FNM,
+	_NAV,
+	_FNC
+};
+
+enum dz65rgb_keycodes {
+	REBOOT = SAFE_RANGE,
+	REEPROM,
+	MAS_CRM,
+	MAS_PRP,
+	MAS_RED,
+	MAS_GRN,
+	MAS_BLU,
+	MAS_CYN,
+	MAS_MGT,
+	MAS_YEL,
+	MAS_KEY,
+	MAS_WHT,
+};
+
+#define TG_NKRO MAGIC_TOGGLE_NKRO
+#define _V_V_V_ KC_TRNS
+#define LT_CAPS LT(_NAV, KC_CAPS)
+extern bool autoshift_enabled;
+#define MT_APP  RALT_T(KC_APP)
+#define LM_LALT LM(_FNM, MOD_LALT)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+	[_QWERTY] = LAYOUT_65_ansi(
+		KC_GESC, KC_1,     KC_2,     KC_3,   KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,     KC_MINS,  KC_EQL,  KC_BSPC, KC_DEL,  \
```

Comment:
The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater without them.

Reply:
I removed them from the templates ;)